src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -6150,14 +6153,26 @@ init_myhook(void)
 	dcache_size_bits = DCACHE_SIZE_BITS;
 	rc               = d_getenv_uint32_t("D_IL_DCACHE_SIZE_BITS", &dcache_size_bits);
 	if (rc != -DER_SUCCESS && rc != -DER_NONEXIST)
-		D_WARN("'D_IL_DCACHE_SIZE_BITS' env variable could not be used: " DF_RC "\n",
-		       DP_RC(rc));
+		DS_WARN(rc, "'D_IL_DCACHE_SIZE_BITS' env variable could not be used");
 
 	dcache_rec_timeout = DCACHE_REC_TIMEOUT;
 	rc                 = d_getenv_uint32_t("D_IL_DCACHE_REC_TIMEOUT", &dcache_rec_timeout);
 	if (rc != -DER_SUCCESS && rc != -DER_NONEXIST)
-		D_WARN("'D_IL_DCACHE_REC_TIMEOUT' env variable could not be used: " DF_RC "\n",
-		       DP_RC(rc));
+		DS_WARN(rc, "'D_IL_DCACHE_REC_TIMEOUT' env variable could not be used");
+
+	dcache_gc_period = DCACHE_GC_PERIOD;
+	rc               = d_getenv_uint32_t("D_IL_DCACHE_GC_PERIOD", &dcache_gc_period);
+	if (rc != -DER_SUCCESS && rc != -DER_NONEXIST)
+		DS_WARN(rc, "'D_IL_DCACHE_GC_PERIOD' env variable could not be used");
+
+	dcache_gc_reclaim_max = DCACHE_GC_RECLAIM_MAX;
+	rc = d_getenv_uint32_t("D_IL_DCACHE_GC_RECLAIM_MAX", &dcache_gc_reclaim_max);
+	if (rc != -DER_SUCCESS && rc != -DER_NONEXIST)
+		DS_WARN(rc, "'D_IL_DCACHE_GC_RECLAIM_MAX' env variable could not be used");
+	if (dcache_gc_reclaim_max <= 0) {
+		D_WARN("'D_IL_DCACHE_GC_RECLAIM_MAX' env variable could not be used: value <= 0.");
+		dcache_gc_reclaim_max = DCACHE_GC_RECLAIM_MAX;
+	}
```

Comment:
The document says "The garbage collector can be deactivated with setting a value of 0 to the `D_IL_DCACHE_GC_RECLAIM_MAX`", so zero looks a valid number here.
dcache_gc_reclaim_max is defined as uint32_t. It can not be negative.

Reply:
Good catch.  I am going to fix this asap

- [ ] Fix `D_IL_DCACHE_GC_RECLAIM_MAX`

Reply:
> Good catch. I am going to fix this asap
> 
> * [x]  Fix `D_IL_DCACHE_GC_RECLAIM_MAX`

Fixed with commit 6c5baf2a2